test_conformance/math_brute_force/main.cpp

```diff
@@ -62,8 +62,6 @@ static int32_t gEndTestNumber = -1;
 int gSkipCorrectnessTesting = 0;
 int gStopOnError = 0;
 static bool gSkipRestOfTests;
-int gMeasureTimes = 0;
```

Comment:
Could we delete `gSimdSize` now too? Can't see anymore uses now the timing code has been removed and it's always hardcoded to 1

Reply:
This is actually part of my next PR. :-)
There are a few other dead globals.

Reply:
nice 😄